pythran/tables.py

```diff
@@ -187,7 +188,7 @@ def update_effects(self, node):
         "clear": MethodIntr(),
         "copy": ConstMethodIntr(),
         "get": ConstMethodIntr(),
-        "has_key": ConstMethodIntr(),
+        "has_key": ConstMethodIntr(return_range=prange.bool_values),
```

Comment:
I wonder if we could state that it returns a bool, so we know it's a ``prange.bool_values``?